app/src/components/SoldArtworkList.tsx

```diff
@@ -37,25 +37,22 @@ class SoldArtworkList extends React.Component<ContractProps, SoldArtworkListStat
     const currentAccount = this.props.accounts[0];
     let eventsFound = 0;
     const soldInfoArray: any[] = [];
+    const filter = {filter: {from: currentAccount}}
+    const options = {filter, fromBlock: 0}
 
     await governance.getPastEvents(
-      'RecordARR',
-      (errors: any, events: any) => {
-        if (!errors) {
-          for (let i = 0; i < events.length; i++) {
-            if (events[i].returnValues.from === currentAccount) {
-              eventsFound += 1;
-              const soldInfo: SoldInformation = {
-                tokenId: events[i].returnValues.tokenId,
-                price: events[i].returnValues.price,
-                newOwner: events[i].returnValues.to,
-              };
-              soldInfoArray.push(soldInfo);
-            }
-          }
+      'RecordARR', options).then(function(events:any)
```

Comment:
i think you should put a '.catch(err: any)' block on this 'then'

Reply:
also i dont think the whitespace is right

Reply:
how so? Happy to change the whitespace

Reply:
/Think I ran lint:fix in the wrong directory